src/Education/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Error displaying Rule Description Window: {0}</value>
   </data>
   <data name="RuleDescriptionToolWindowCaption" xml:space="preserve">
-    <value>Rule Description</value>
+    <value>SonarLint Rule Description</value>
```

Comment:
Minor tweak : might as well make it clear it our tool window.

I'll create a separate ticket to look at adding an icon to the tool window.